code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -166,6 +173,8 @@
 	load_method = 2
 	var/cover_open = 0
 
+/obj/item/weapon/gun/projectile/automatic/l6_saw/isHandgun()
```

Comment:
The description of this thing says it's a LMG but you say you're making SMGs count as handguns.
t. can't into guns

Reply:
`return FALSE	`, this gun is unaffected

every automatic is a child of the normal R&D SMG for some god awful reason, but that's the one i'm changing

Reply:
oh, my bad